scripts/deploy-backend.sh

```diff
@@ -50,10 +50,11 @@ update_backend()
 }
 
 bind_backend_to_services() {
-    cf bind-service "$CGHOSTNAME_BACKEND" tdp-django-static-sandbox
-    cf bind-service "$CGHOSTNAME_BACKEND" tdp-storage-sandbox
-    cf bind-service "$CGHOSTNAME_BACKEND" tanf-storage
-    cf bind-service "$CGHOSTNAME_BACKEND" tdp-db
+    env=${$CF_SPACE"#tanf-"}
+    cf bind-service "$CGHOSTNAME_BACKEND" "tdp-static-sandbox-${env}"
+    cf bind-service "$CGHOSTNAME_BACKEND" "tdp-storage-sandbox-${env}"
+    cf bind-service "$CGHOSTNAME_BACKEND" "tanf-storage-${env}"
+    cf bind-service "$CGHOSTNAME_BACKEND" "tdp-db-${env}"
```

Comment:
We suffix the services with the environment they're being deployed to now, to account for dev, staging, and higher environments in the future.